apps/dsa.c

```diff
@@ -221,14 +222,14 @@ int dsa_main(int argc, char **argv)
         EVP_PKEY_set1_DSA(pk, dsa);
         if (outformat == FORMAT_PVK) {
             if (pubin) {
-                BIO_printf(bio_err, "PVK form impossible with public key input\n");
+                BIO_printf(bio_err,
+                           "PVK form impossible with public key input\n");
                 EVP_PKEY_free(pk);
                 goto end;
             }
             assert(private);
             i = i2b_PVK_bio(out, pk, pvk_encr, 0, passout);
-        }
-        else if (pubin || pubout)
+        } else if (pubin || pubout)
             i = i2b_PublicKey_bio(out, pk);
```

Comment:
If it were me, I'd wrap this one in { and } as well, just for consistency with the rest of the `if` structure